Adding Text-based NP Enrichment (TNE) dataset (#4153)
huggingface/datasets: datasets/tne/README.md

---
annotations_creators:
- crowdsourced
language_creators:
- found
languages:
- en
licenses:
- mit
multilinguality:
- monolingual
pretty_name: Text-based NP Enrichment
size_categories:

> - unknown

Reviewer: ```suggestion
- 1K<n<10K
```

Author: Can't believe I missed that.
Btw, it's a bit more nuanced than that. We have in the dataset about 5K documents, but there are many more annotations. What is the standard here? To report the number of documents or annotations?

Reviewer: We report the number of samples (i.e. the elements that we `yield`), which here correspond to the documents

source_datasets:
- original
task_categories:
- text-retrieval
task_ids:
- document-retrieval
---

> # Dataset Card for tne

Reviewer: ```suggestion
# Dataset Card for Text-based NP Enrichment
```

  - [Data Instances](#data-instances)
  - [Data Fields](#data-instances)
  - [Data Splits](#data-instances)
- [Dataset Creation](#dataset-creation)
  - [Curation Rationale](#curation-rationale)
  - [Source Data](#source-data)
  - [Annotations](#annotations)
  - [Personal and Sensitive Information](#personal-and-sensitive-information)
- [Considerations for Using the Data](#considerations-for-using-the-data)
  - [Social Impact of Dataset](#social-impact-of-dataset)
  - [Discussion of Biases](#discussion-of-biases)
  - [Other Known Limitations](#other-known-limitations)
- [Additional Information](#additional-information)
  - [Dataset Curators](#dataset-curators)
  - [Licensing Information](#licensing-information)
  - [Citation Information](#citation-information)

## Dataset Description

- **Homepage:** https://yanaiela.github.io/TNE/
- **Repository:** https://github.com/yanaiela/TNE
- **Paper:** https://arxiv.org/abs/2109.12085
- **Leaderboard:** [TNE](https://leaderboard.allenai.org/tne/submissions/public)
[TNE OOD](https://leaderboard.allenai.org/tne-ood/submissions/public)
- **Point of Contact:** [Yanai Elazar](mailto:yanaiela@gmail.com)

### Dataset Summary

> TNE is an NLU task, which focus on relations between noun phrases (NPs) that can be mediated via prepositions. The dataset contains 5,497 documents, annotated exhaustively with all possible links between the NPs in each document.

Reviewer: ```suggestion
Text-based NP Enrichment (TNE) is a natural language understanding (NLU) task, which focus on relations between noun phrases (NPs) that can be mediated via prepositions. The dataset contains 5,497 documents, annotated exhaustively with all possible links between the NPs in each document.
```

> The main data comes from WikiNews, which is used for train/dev/test. We also collected an additional set of 509 documents to serve as OOD, from the Book Corpus, IMDB reviews and Reddit.

Reviewer: ```suggestion
The main data comes from WikiNews, which is used for train/dev/test. We also collected an additional set of 509 documents to serve as out of distribution (OOD) data points, from the Book Corpus, IMDB reviews and Reddit.
```

### Supported Tasks and Leaderboards

[Needs More Information]

### Languages

The text in the dataset is in English, as spoken in the different domains we include. The associated BCP-47 code is `en`.

## Dataset Structure

### Data Instances

> Each file is in a jsonl format, containing a dictionary of a single document, in each line.

Reviewer: ```suggestion
The original files are in a jsonl format, containing a dictionary of a single document, in each line.
```

Additional details on the data statistics can be found in the [paper](https://arxiv.org/abs/2109.12085)

## Dataset Creation

### Curation Rationale

TNE was build as a new task for language understanding, focusing on extracting relations between nouns, moderated by prepositions.

### Source Data

#### Initial Data Collection and Normalization

[Needs More Information]

#### Who are the source language producers?

[Needs More Information]

### Annotations

#### Annotation process

[Needs More Information]

#### Who are the annotators?

[Needs More Information]

### Personal and Sensitive Information

[Needs More Information]

## Considerations for Using the Data

### Social Impact of Dataset

[Needs More Information]

### Discussion of Biases

[Needs More Information]

### Other Known Limitations

[Needs More Information]

## Additional Information

### Dataset Curators

The dataset was created by Yanai Elazar, Victoria Basmov, Yoav Goldberg, Reut Tsarfaty, during work done at Bar-Ilan University, and AI2.

### Licensing Information

The data is released under the MIT license.

### Citation Information

> @article{tne,
>     author = {Elazar, Yanai and Basmov, Victoria and Goldberg, Yoav and Tsarfaty, Reut},
>     title = "{Text-based NP Enrichment}",
>     journal = {Transactions of the Association for Computational Linguistics},
>     year = {2022},
> }

Reviewer: ````suggestion
```bibtex
@article{tne,
    author = {Elazar, Yanai and Basmov, Victoria and Goldberg, Yoav and Tsarfaty, Reut},
    title = "{Text-based NP Enrichment}",
    journal = {Transactions of the Association for Computational Linguistics},
    year = {2022},
}
```
````

### Contributions

> The first author added this dataset.

Reviewer: ```suggestion
Thanks to [@yanaiela](https://github.com/yanaiela), who is also the first author of the paper, for adding this dataset.
```

Author: Do I really need to thank myself? 😅

Reviewer: this is a thank on behalf of the hugging face community ;)

  * `complement`: the id of the complement np
  * `preposition`: the preposition that links between the anchor and the complement. This can take one out of 24 pre-defined preposition (23 + member(s)-of)
  * `complement_coref_cluster_id`: the coreference id, which the complement is part of.
* `coref`: the coreference labels. It contains a list of dictionaries, where each dictionary contains:
  * `id`: the id of the coreference cluster
  * `members`: the ids of the nps members of such cluster
  * `np_type`: the type of cluster. It can be either 
    * `standard`: regular coreference cluster
    * `time/date/measurement`: a time / date / measurement np. These will be singletons.
    * `idiomatic`: an idiomatic expression
* `metadata`: metadata of the document. It contains the following:
  * `annotators`: a dictionary with anonymized annotators id
    * `coref_worker`: the coreference worker id
    * `consolidator_worker`: the consolidator worker id
    * `np-relations_worker`: the np relations worker id
  * `url`: the url where the document was taken from (not always existing)
  * `source`: the original file name where the document was taken from

### Data Splits

> The dataset is spread across four files, for the four different splits: train, dev, test and ood.

Reviewer: ```suggestion
The dataset is spread across four files, for the four different splits: train, dev, test and test_ood.
The `np_relations` field is an empty list for the test and test_ood splits.

```
